.circleci/config.yml

```diff
@@ -11,10 +11,11 @@ jobs:
       - save_cache:
           key: dependency-cache-{{ checksum "pom.xml" }}
           paths: ~/.m2
+      - run: mvn -X scoverage:report
```

Comment:
Does this do the build as well internally?

Reply:
Yes Santhosh, Just this will compile and executes the test cases.